examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -125,6 +240,22 @@ void LightSwitchMgr::TriggerLightSwitchAction(LightSwitchAction action, bool isG
     DeviceLayer::PlatformMgr().ScheduleWork(SwitchWorkerFunction, reinterpret_cast<intptr_t>(data));
 }
 
+void LightSwitchMgr::TriggerLevelControlAction(uint8_t stepMode, bool isGroupCommand)
```

Comment:
I'm not sure i understand what we are trying to achieve - is this a long press move level ?

Reply:
I'm not sure i agree with the idea of adding another functionnality to the button event handler...
@jmartinez-silabs do you think it is worth while to implement a "toogle" feature on button 1?

Reply:
I've modified the button logic so that the level of the bounded app can be controlled by using buttons, similar to how we do tilt and lift operations in window app on button press events

Reply:
@mkardous-silabs Yeah we have something similar with window-app button.
If it is documented what the buttons do.
I'm ok with implementing the long press on both buttons